features/shared/checkout_previous_branch_after_success/deleted_current_branch/previous_branch_deleted.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Allow checking out the correct previous Git branch after running a Git Town commmand that deletes the previous and current branches
+
+  (see ../same_current_branch/previous_branch_same.feature)
+
+
+  Scenario: checkout previous branch after a git-prune-branches that deletes previous and current branches
+    Given I have feature branches named "previous" and "current"
+    And I am on the "previous" branch
+    And I checkout the "current" branch
```

Comment:
How do you feel with expressing the fact that "previous" is the previous Git branch directly here? That's what we try to do in Cucumber.

> Given I have feature branches named "previous" and "current"
> And I am on the "current" branch with "previous" as the previous Git branch
> When I run `git prune-branches`